sos/report/plugins/aap_receptor.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 Rudnei Bertol Jr <rudnei@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import glob
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPreceptorPlugin(Plugin, RedHatPlugin):
+    short_desc = 'AAP receptor plugin'
+    plugin_name = 'aap_receptor'
+    profiles = ('sysmgmt', 'ansible')
+    packages = ('receptor', 'receptorctl')
+    services = ('receptor',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/receptor",
+            "/var/lib/receptor",
+        ])
+
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/receptor"
+            ])
+        else:
+            self.add_copy_spec([
+                "/var/log/receptor/*.log"
+            ])
+
+        self.add_forbidden_path([
+            "/etc/receptor/tls",
+            "/etc/receptor/*key.pem"
+        ])
+
+        self.add_cmd_output([
+            "ls -ll /etc/receptor",
+            "ls -ll /var/run/receptor",
+            "ls -ll /var/run/awx-receptor"
+        ])
```

Comment:
Typically for `ls` collections we'll use `-alhR`. Are we not interested in recursive listing here? Do we need to consider `-Z` for selinux labeling? (On that point, note to self: we really need to look at if we want a wrapper for dir listing collections).

Reply:
I liked the idea for the selinux, it will help us in case of mislabeling. Working on this change as well.